### PR TITLE
[Issue-21] Enforce precision in bridge function

### PR DESCRIPTION
**Pull request by pheuberger**
Addresses issue #21

### CODE REVIEW

**Comment by pheuberger** [comment]
is there a way to ignore a rule in slither @kargakis?

File: contracts/ToucanRegenBridge.sol [hunk 1]
```diff
@@ -112,6 +112,9 @@ contract ToucanRegenBridge is Ownable, Pausable {
         require(toucanContractRegistry.checkERC20(tco2), "not a TCO2");
         require(nctPool.checkEligible(tco2), "TCO2 not eligible for NCT pool");
 
+        uint256 precisionTest = (amount / 10**12) * 10**12;
```
Comment (kargakis) [inline contracts/ToucanRegenBridge.sol:115]: We can avoid making the exponentiations in the runtime by leveraging a `constant` `1e12`

File: contracts/ToucanRegenBridge.sol [hunk 2]
```diff
@@ -112,6 +112,9 @@ contract ToucanRegenBridge is Ownable, Pausable {
         require(toucanContractRegistry.checkERC20(tco2), "not a TCO2");
         require(nctPool.checkEligible(tco2), "TCO2 not eligible for NCT pool");
```
Comment (kargakis) [inline contracts/ToucanRegenBridge.sol:114]:
```suggestion
//slither-disable-next-line divide-before-multiply
```

**Comment by pheuberger** [comment]
> > is there a way to ignore a rule in slither @kargakis?
> 
> Yes, I have left a comment inline, `//slither-disable-next-line` is the start of the comment and docs of the rule (which is linked by Slither in logs) point to the end of the comment that is needed, [`divide-before-multiply`](https://github.com/crytic/slither/wiki/Detector-Documentation#divide-before-multiply) in this case.

awesome, thank you!

**Review by kargakis — APPROVED** [review]

**Pull request merged**